test/python/transpiler/test_calibrationbuilder.py

```diff
@@ -355,7 +492,142 @@ def test_non_native_cr(self):
             builder.call_gate(SXGate(), qubits=(1,))
             builder.call_gate(RZGate(np.pi / 2), qubits=(1,))
 
-        self.assertEqual(schedule(test_qc, self.backend), target_qobj_transform(ref_sched))
+        self.assertEqual(schedule(test_qc, backend), target_qobj_transform(ref_sched))
+
+    # # TODO - uncomment this when pulse builder works with BackendV2
```

Comment:
Please remove the code itself. You can keep it locally, or make another PR with on hold tag.

Reply:
I'll make another PR once this one is merged.